lib/CXGN/Onto.pm

```diff
@@ -130,8 +139,16 @@ sub store_composed_term {
     my $schema = $self->schema();
     my $dbh = $schema->storage->dbh;
 
-    my $contains_relationship = $schema->resultset("Cv::Cvterm")->find({ name => 'contains' });
-    my $variable_relationship = $schema->resultset("Cv::Cvterm")->find({ name => 'VARIABLE_OF' });
+    my $relationship_cv = $schema->resultset("Cv::Cv")->find({ name => 'relationship'});
+    my $rel_cv_id;
+    if ($relationship_cv) {
+        $rel_cv_id = $relationship_cv->cv_id ;
+    } else {
+        print STDERR "relationship ontology is not found in the database\n";
```

Comment:
If $relationship_cv is not defined, is it supposed to continue anyway? Would the the query at line 151 function with undefined $rel_cv_id?